src/commands/status/docker.js

```diff
@@ -0,0 +1,99 @@
+const { table } = require('table');
+const chalk = require('chalk');
+
+const ContainerIsNotPresentError = require('../../docker/errors/ContainerIsNotPresentError');
+
+const BaseCommand = require('../../oclif/command/BaseCommand');
+
+const PRESETS = require('../../presets');
+
+class DockerStatusCommand extends BaseCommand {
+  /**
+   * @param {Object} args
+   * @param {Object} flags
+   * @param {DockerCompose} dockerCompose
+   * @return {Promise<void>}
+   */
+  async runWithDependencies(
+    {
+      preset,
+    },
+    flags,
+    dockerCompose,
+  ) {
+    const serviceHumanNames = {
+      core: 'Core',
+      sentinel: 'Sentinel',
+    };
+
+    if (preset !== 'testnet') {
+      Object.assign(serviceHumanNames, {
+        drive_mongodb_replica_init: 'Initiate Drive MongoDB replica',
+        drive_mongodb: 'Drive MongoDB',
+        drive_abci: 'Drive ABCI',
+        drive_tendermint: 'Drive Tendermint',
+        dapi_insight: 'DAPI Insight',
+        dapi_api: 'DAPI API',
+        dapi_tx_filter_stream: 'DAPI Transactions Filter Stream',
+        dapi_envoy: 'DAPI Envoy',
+        dapi_nginx: 'DAPI Nginx',
+      });
+    }
+
+    const tableRows = [
+      ['Service', 'ID', 'Status'],
```

Comment:
It would be very useful to add versions here. We could add LABEL to docker images with version and then get it back here via inspect.

Reply:
I've added functionality to support version numbers from labels. The key is read from the `org.dash.version` key, that should be fine right? I've added an [example label to my test image of Sentinel](https://github.com/strophy/docker-sentinel/commit/1aecef0fa02a03f78d389b17632c8871a5ef2c20), does that look right? Should the container track the version of the software it contains? Did you want to add these labels in the Dockerfile or in Docker Compose?

Reply:
A container doesn’t need to track it, I suppose. Yeah, let’s just put versions to service Docker files.

Reply:
So the container should have a separate version, like 0.1.0? Or same version as Sentinel, like 1.4.0?

Reply:
Yeah, I thought it will be a software version that we run inside a container.

Reply:
OK, I'll push and build a version for Sentinel.

Reply:
Let’s do that in the next iteration (separate PR)?

Reply:
Versions are already done in 2629b6d - I think we can merge this once conflicts are resolved?